ci/jenkins/pipelines/skuba-e2e-nightly.Jenkinsfile

```diff
@@ -5,11 +5,9 @@
 pipeline {
    agent { node { label 'caasp-team-private' } }
 
-   properties([
-       parameters([
-           string(name: 'E2E_MAKE_TARGET_NAME', defaultValue: 'all', description: 'The make target to run (only e2e related)')
-       ])
-   ])
+   parameters {
+        string(name: 'E2E_MAKE_TARGET_NAME', defaultValue: 'all', description: 'The make target to run (only e2e related)')
```

Comment:
below you use `e2e` hard coded. i'm not expert in this, just noticed this.

Reply:
yes, the groovy linter says:

```
WorkflowScript: 28: Expected string literal @ line 28, column 15.
           stage("Run Skuba ${E2E_MAKE_TARGET_NAME} Test") {
                 ^
```

which basically means I can't use a variable in the stage description, but it's not a big deal so i hardcoded just e2e. we have the 'E2E_MAKE_TARGET_NAME' in the label, so it shows up in the UI